### DISCUSSION TITLE
Welcome to pwa-bundle Discussions!

### DISCUSSION THREAD

**Discussion by Spomky** [Spomky-Labs/pwa-bundle#96] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by davidromani** [comment]
Hi, I'm a full stack developer during last 15 years. I usually work with Symfony backend features but sometimes my customers needs superpowerfull rich frontend features.

As far as I've discovered Symfony UX and Hotwire (Turbo + Stimulus + Strada) technologies I've even more interested doing awesome live behaviours.

Recently I must implement and offline complex feature for my last project and I'm giving a try this bundle to handle Service Workers. For now I need some help beacuse I'm really stuck to achieve this feature.

Hope to found somebody here to help me or discover new hints to achieve my goal.

**Comment by tacman** [comment]
You're in the right place, and at just the right time.

@Spomky is giving a talk in a few weeks at the Symfony Paris conference, so development on this bundle is pretty active.  I imagine so that once this gets in front of a bunch of Symfony developers, it's going to become very popular.

**Comment by Spomky** [comment]
Hi David,

This bundle is quite new and lots of exciting features come as it evolves.
Feedback, thoughts, ideas or any comment you may have are welcome.
I will do my best to help you.
